guacamole-ext/src/main/java/org/apache/guacamole/net/auth/ActivityRecordSet.java

```diff
@@ -19,7 +19,9 @@
 
 package org.apache.guacamole.net.auth;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.List;
```

Comment:
Same here - unless the build is currently failing, it's unlikely new `import` statements are needed without other code changes within the same file requiring them.